db/db/src/main/java/io/helidon/db/DbExecute.java

```diff
@@ -0,0 +1,407 @@
+     *
+     * @param statement  the query statement to be executed
+     * @param parameters query parameters to set
+     * @return database query execution result which can contain multiple rows
+     */
+    default DbRowResult<DbRow> query(String statement, Object... parameters) {
+        return createQuery(statement).params(parameters).execute();
+    }
+
+    /*
+     * GET
+     */
+
+    /**
+     * Create a database query returning a single row using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return database statement that can process query returning a single row
+     */
+    DbStatement<?, CompletionStage<Optional<DbRow>>> createNamedGet(String statementName, String statement);
+
+    /**
+     * Create a database query returning a single row using a statement defined in the configuration file.
+     *
+     * @param statementName the name of the configuration node with statement
+     * @return database statement that can process query returning a single row
+     */
+    DbStatement<?, CompletionStage<Optional<DbRow>>> createNamedGet(String statementName);
+
+    /**
+     * Create a database query returning a single row using a statement passed as an argument.
+     *
+     * @param statement the query statement to be executed
+     * @return database statement that can process query returning a single row
+     */
+    DbStatement<?, CompletionStage<Optional<DbRow>>> createGet(String statement);
+
+    /**
+     * Create and execute a database query using a statement defined in the configuration file.
+     *
+     * @param statementName the name of the configuration node with statement
+     * @param parameters    query parameters to set
+     * @return database query execution result which can contain single row
+     */
+    default CompletionStage<Optional<DbRow>> namedGet(String statementName, Object... parameters) {
+        return createNamedGet(statementName).params(parameters).execute();
+    }
+
+    /**
+     * Create and execute a database query using a statement passed as an argument.
+     *
+     * @param statement  the query statement to be executed
+     * @param parameters query parameters to set
+     * @return database query execution result which can contain single row
+     */
+    default CompletionStage<Optional<DbRow>> get(String statement, Object... parameters) {
+        return createGet(statement).params(parameters).execute();
+    }
+
+    /*
+     * INSERT
+     */
+
+    /**
+     * Create an insert statement using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return database statement that can insert data
+     */
+    default DbStatement<?, CompletionStage<Long>> createNamedInsert(String statementName, String statement) {
```

Comment:
Another occurence of _hybrid_ method with both statement name and statement string.